packages/admin-panel-server/src/app/createApp.ts

```diff
@@ -47,8 +58,22 @@ export function createApp() {
   const app = new OrchestratorApiBuilder(new TupaiaDatabase())
     .useSessionModel(AdminPanelSessionModel)
     .verifyLogin(hasTupaiaAdminPanelAccess)
-    .get('/v1/user', handleWith(UserRoute))
-    .get('/v1/hierarchy/:hierarchyName/:entityCode', handleWith(FetchHierarchyEntitiesRoute))
+    .get('/v1/user', verifyBESAdminAccess, handleWith(UserRoute))
```

Comment:
Should this `user` route be using `BESAdminAccess` privileges now?

Reply:
This one is kinda only used for viz-builder at the moment. But I think you're right, there's no reason why it couldn't be use by the general admin-panel app in the future